recipes/dpu-utils/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "dpu-utils" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dpu_utils-{{ version }}.tar.gz
+  sha256: d385083cb6dde43d9f469dc8627dfd13f25d95789890b51b55f90ebeefa15311
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  skip: True  # [py>37]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - azure-identity
+    - azure-storage-blob
+    - chardet
+    - idna
+    - isodate
+    - msal
+    - msrest
+    - oauthlib
+    - portalocker
+    - pycparser
+    - cryptography
+    - cffi
+    - docopt
+    - numpy
+    - python
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
Since `noarch` doesn't work for this package (I am not entirely sure why), this cannot be used.

Reply:
Resolving conversation after https://github.com/conda-forge/staged-recipes/pull/15363/commits/ab09e11bf492a222ad6545f010ac0e34f5596f5f